polaris-react/tsconfig.json

```diff
@@ -3,13 +3,13 @@
   "compilerOptions": {
     "composite": true,
     "emitDeclarationOnly": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
There were a lot of build errors related to this.

Reply:
Yep had a quick poke at it, It's because the codemod doesn't handle "you should use `import type` if you're only importing types. My fork doesn't help with that either.

`importsNotUsedAsValues: error` basically enforces "if you only use types from an import then it must use `import type`. Previously `import React, {SomeType} from 'react'` wasn't a violation - you used React at runtime so the rule that that's ok. With the automatic runtime you don't need the default export anymore, so the codemod transforms that code into `import {SomeType} from 'react'`. And then typescript shouts at you because you're only accessing types but you're using `import`, not `import type`.

You've got two approaches here:

The low-key just-get-it-out-approach: Manually go adjust those `import {SomeType} from 'react'` lines that TS complains about to `import type {SomeType} from 'react'`. There's like 11 violating files that seems pretty easy if you want to style it out in this PR.

The somewhat sweatier approach: Go enable the [`@typescript-eslint/consistent-type-imports`](https://typescript-eslint.io/rules/consistent-type-imports) and `@typescript-eslint/consistent-type-exports` lint rules on the main branch, then rebase the v11 branch, then rebase this branch. These are autofixable rules that will force usage of `import type` when you import a type. This is actually a little stricter than the `"importsNotUsedAsValues": "error",` stuff described above - the linter will change the existing usage of `import React, {SomeType} from 'react'` to be `import React from 'react'; `import type {SomeType} from 'react'`. See https://typescript-eslint.io/blog/consistent-type-imports-and-exports-why-and-how. If you do this, then get this branch rebased atop of that change, then running `eslint --fix`  will pull them into a shape that will keep tsc happy